core/new-gui/src/app/workspace/service/execute-workflow/mock-workflow-plan.ts

```diff
@@ -16,20 +16,20 @@ export const mockWorkflowPlan_scan_result: WorkflowGraph = new WorkflowGraph(
 
 export const mockLogicalPlan_scan_result: LogicalPlan = {
   operators: [
-    {
-      ...mockScanPredicate.operatorProperties,
-      operatorID: mockScanPredicate.operatorID,
-      operatorType: mockScanPredicate.operatorType,
-      inputPorts: mockScanPredicate.inputPorts,
-      outputPorts: mockScanPredicate.outputPorts,
-    },
     {
       ...mockResultPredicate.operatorProperties,
       operatorID: mockResultPredicate.operatorID,
       operatorType: mockResultPredicate.operatorType,
       inputPorts: mockResultPredicate.inputPorts,
       outputPorts: mockResultPredicate.outputPorts,
     },
+    {
```

Comment:
seems unnecessary?

Reply:
the order unfortunately matters as it is stored as an array. so this change is needed with the new way of finding the subDAG.